iidm/iidm-api/src/main/java/com/powsybl/iidm/network/ShuntCompensatorNonLinearModel.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network;
+
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * @author Miora Ralambotiana <miora.ralambotiana at rte-france.com>
+ */
+public interface ShuntCompensatorNonLinearModel extends ShuntCompensatorModel {
+
+    interface Section {
+
+        /**
+         * Get the susceptance in S of the section.
+         */
+        double getB();
+
+        /**
+         * Get the conductance in S of the section.
+         */
+        double getG();
+    }
+
+    /**
+     * Get an optional of the section associated with a given section number if it exists.
+     * If such a section does not exist, return an empty optional.
+     *
+     */
+    Optional<Section> getSection(int sectionIndex);
```

Comment:
I would model section in IIDM as a [0, maxSectionCount[ array and here avoid the optional.